sql/ddl_api.sql

```diff
@@ -17,7 +17,7 @@ CREATE OR REPLACE FUNCTION  create_hypertable(
     number_partitions       INTEGER = NULL,
     associated_schema_name  NAME = NULL,
     associated_table_prefix NAME = NULL,
-    chunk_time_interval     BIGINT = NULL,
+    chunk_time_interval     anyelement = NULL::bigint,
```

Comment:
Why is the null cast to bigint here ?

Reply:
since the type is anyelement you have to let it know the type of the default element